uui-components/src/inputs/__tests__/Checkbox.test.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Checkbox, CheckboxProps } from '../Checkbox';
+import { render, screen, fireEvent, setupComponentForTest } from '@epam/uui-test-utils';
+
+async function setupCheckbox(params: Partial<CheckboxProps>) {
+    const { mocks, setProps } = await setupComponentForTest<CheckboxProps>(
+        (context) => ({
+            value: params.value,
+            onValueChange: params.onValueChange || jest.fn().mockImplementation((newValue) => {
+                context.current.setProperty('value', newValue);
+            }),
+            ...params,
+        }),
+        (props) => {
+            return <Checkbox { ...props } />;
+        },
+    );
+    return {
+        setProps,
+        mocks,
+    };
+}
+
+describe('Checkbox', () => {
```

Comment:
Let's also check that click on label toggle checkbox value and that user can change value via keyboard